cli/cmd/bootstrap.go

```diff
@@ -7,7 +7,6 @@ import (
 	"github.com/chanzuckerberg/happy/shared/hclmanager"
 	"github.com/chanzuckerberg/happy/shared/util"
 	"github.com/pkg/errors"
-	"github.com/sirupsen/logrus"
 	log "github.com/sirupsen/logrus"
```

Comment:
1.21 introduced the `slog` package. I want to start using that going forward. Just a TODO for later

https://go.dev/blog/slog